internal/config/config.go

```diff
@@ -3,41 +3,126 @@ package config
 import (
 	"errors"
 	"os"
+	"strconv"
 )
 
 type Config struct {
 	JWTTokenSecret string
 	RestPort       string
 	GrpcPort       string
 	Address        string
+	DB
+}
+
+type DB struct {
```

Comment:
Consider, that instead of generalist "DB" specify database, e.g. Postgres, Mongo, etc. It's also a better approach, even if you use only one database, as you can extend application with another DB in the future (e.g. Redis for caching, Elastic for optimized text search, etc.)

Reply:
fixed